seeds/perl_seed.pl

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env perl
+use strict;
+use warnings;
+use Getopt::Long;
+use Pod::Usage;
+
+# Perl script seed.
+# Use this to get started quickly, and customize it to your needs.
+
+my $type = 'flower';
+my $man = 0;
+my $help = 0;
+my $water = 0;
+GetOptions( 'help|h' => \$help, 
+            'man' => \$man,
+            'water|w' => \$water,
+          );
+pod2usage(1) if $help;
+pod2usage(-exitstatus => 0, -verbose => 2) if $man;
+
+# Write your code below
+if ($water){
+  water_the_plant(${type}) ;
+}
+else {
+  plant_a_seed(${type});
+}
+
+sub plant_a_seed {
+  my $seed = shift;
+  print "You planted a ${seed} seed!\n";
+}
+
+sub water_the_plant {
+  my $plant = shift;
+  print "You watered the ${plant}.\n";
+}
+
+# Write your perldoc below the __END__
+__END__
+
+=head1 NAME
+sample - Using Getopt::Long and Pod::Usage
+
+=head1 SYNOPSIS
+sample [options] [file ...]
+  Options:
```

Comment:
You may need to include the word `usage` in the output here to get the tests to pass.

Reply:
ok thank, 
I add it very soon.

regards